Lookuptable/requires.txt

```diff
@@ -0,0 +1,15 @@
+## Stores information about all the libraries, modules, and packages that are used in this project.
+
+flake8>=3.8.4
```

Comment:
Are we using the library `flake8` and `matplotlib` here?

Reply:
We are not using it. So I have updated and pushed the code. Commit id : d09e3f95d7bc85be69e2ee9de0612cc5851ba023